test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -1,5 +1,7 @@
 using namespace System.Diagnostics
 
+Import-Module $PSScriptRoot\..\Common\Test.Helpers.psm1
```

Comment:
It would be nice if this could be less fragile to directory structure - you couldn't move `Test.Helpers.psm1` without making many changes.  On the other hand, one obvious alternative `Import-Module $TestHelpers` or something like that would require running something first, like `Start-PSPester`, which we don't need to do today, you can just run the test directly, like `.\Foo.Tests.ps1`.

It would be even nicer if it was not necessary.

If `$env:PSModulePath` is set correctly and the module is a more proper module (in a directory named `Test.Helpers`), then we could rely on module auto-loading to load the module, except maybe in a test specific to module auto-loading.